tests/spread/plugins/craft-parts/python-symlinks/task.yaml

```diff
@@ -0,0 +1,43 @@
+summary: >-
+  Verify python symlinks with different base and confinement
+  configurations
+
+environment:
+  # base core22, strict confinement: use provisioned if available
+  SNAP/strict_not_provisioned: strict-not-provisioned
+  SNAP/strict_provisioned: strict-provisioned
+
+  # base core22, classic confinement: requires provisioning
+  SNAP/classic_not_provisioned: classic-not-provisioned
+  SNAP/classic_provisioned: classic-provisioned
+
+  # base core22, devmode: same as strict
+  SNAP/devmode_not_provisioned: devmode-not-provisioned
+  SNAP/devmode_provisioned: devmode-provisioned
+
+  # base bare: requires provisioning
+  SNAP/bare_not_provisioned: bare-not-provisioned
+  SNAP/bare_provisioned: bare-provisioned
+
+  # provisioned from a different part
+  SNAP/provisioned_from_another_part: provisioned-from-another-part
+
+  # using the gnome extension
+  SNAP/strict_gnome_extension: strict-gnome-extension
+
+restore: |
+  cd "${SNAP}"
+  snapcraft clean
+  rm -f ./*.snap
+
+execute: |
+  cd "${SNAP}"
+
+  if [ -f expected-symlink ]; then
```

Comment:
I don't understand how this would work with not provisioned corner cases

Reply:
Could you provide an example of such a corner case?